install/resources/mybb_theme.xml

```diff
@@ -14359,5 +14356,10 @@ $(function() {
 </tr>]]></template>
 		<template name="task_image" version="1800"><![CDATA[<img src="{$mybb->settings['bburl']}/task.php" width="1" height="1" alt="" />]]></template>
 		<template name="smilie" version="1800"><![CDATA[<img src="{$smilie['image']}" alt="{$smilie['name']}" title="{$smilie['name']}" class="smilie smilie_{$smilie['sid']}{$extra_class}"{$onclick} />]]></template>
+	<template name="delete_attachments_button" version="1825"><![CDATA[		
```

Comment:
Sorry to be late to the hall, but this button belongs to `User Control Panel Templates` Group and hence should be named  with prefix `usercp_` otherwise this template will land into some other place / group.

Reply:
Please open a PR to fix this @effone

Reply:
OK. I will address this with the Attachment PR for #4084 I am currently working on.

Reply:
Making a sidenote for self...
This new template should be cached in `usercp.php` as well.